lib/createDIContainer.js

```diff
@@ -139,7 +142,13 @@ async function createDIContainer(options) {
    * Register common services
    */
   container.register({
-    logger: asFunction(() => new Logger(console)).singleton(),
+    logger: asFunction((loggingLevel) => pino({
+      level: loggingLevel,
+      prettyPrint: {
+        ignore: ['debug', 'trace'].includes(loggingLevel) ? '' : 'stack',
```

Comment:
It adds stack for any loglevels or just for errors?

Reply:
Just for `debug` and `trace`.

Reply:
I mean it adds it only when you call `logger.error` or `logger.info` as well?

Reply:
Any method, but when you pass instance of an `Error` class

Reply:
Maybe then we keep it?